### DISCUSSION TITLE
Network devices cannot be used after the router is shared

### DISCUSSION THREAD

**Discussion by yi250** [Mygod/VPNHotspot#266] [Q&A]
It seems to be the underlying limitation of Android. After the network was shared to the router, two IPs were detected, but only one device could be used, and then it would automatically block no device IP, but there were indeed devices under the router network。This probably means that the device network connection under the device is not supported

**Comment by yi250** [comment]
I'm just asking. I hope you'll give it a try 

ε(*´･ω･)з

**Comment by Mygod** [comment]
Sounds like a vendor specific issue since I could not reproduce this. What is your device model and ROM?

**Reply by yi250** [reply]
MIUI
